### ISSUE TITLE
Possible to add layers?

### ISSUE THREAD

**Issue by BoxSo** [StephanGeorg/staticmaps#48]
Let's say i want to add another layer from a tile-Server (http://openfiremap.org/hytiles/z/x/y.png). Would this be possible somehow?

Regards

BoxSon

**Comment by StephanGeorg** [comment]
Sure, but it's a basemap not a layer:

```
const StaticMaps = require('staticmaps');
const options = {
  width: 600,
  height: 400,
  tileUrl: 'http://openfiremap.org/hytiles/{z}/{x}/{y}.png',
};

const map = new StaticMaps(options);
```

**Comment by BoxSo** [comment]
This does not seem to work. All i get is a black image. Maybe i'm doing something wrong ?

**Comment by StephanGeorg** [comment]
Can I see a code snippet?

**Comment by BoxSo** [comment]
![single-marker](https://user-images.githubusercontent.com/3858285/96141721-9559b400-0f01-11eb-86ba-eeb7cbd21ad3.png)

This is the resulting image

**Comment by StephanGeorg** [comment]
Ok, I checked the tiles and its indeed a layer and not a basemap: 
http://openfiremap.org/hytiles/11/1084/700.png

Sorry, I thought the hydrant icons are rendered onto the tile images. This is a new use case and will add a feature to support that kind of "layers".

**Comment by BoxSo** [comment]
Thanks Stephan! appreciate!

Regards

Jens

**Comment by Semurak** [comment]
Is there any news regarding this feature? 

Thanks!

**Comment by JorgenPhi** [comment]
Added a PR for the modifications I made to add 2 tile servers: https://github.com/StephanGeorg/staticmaps/pull/68

**Comment by StephanGeorg** [comment]
Closed with #68
